pipelines/rbsa/rbsa.py

```diff
@@ -80,69 +80,117 @@ def generate_result_plots(self):
         
         df =  adm.load_data([
             { 'name': 'normal_loadshapes.csv', 'read_type': SupportedFileReadType.DATA },
-            { 'name': 'total_loadshapes.csv', 'read_type': SupportedFileReadType.DATA }
+            { 'name': 'enduse_loadshapes.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'total_loadshapes.csv', 'read_type': SupportedFileReadType.DATA },
+            { 'name': 'loadshapes.csv', 'read_type': SupportedFileReadType.DATA }
         ])
 
-        normal_loads = df['normal_loadshapes.csv']
+        normal_loadshapes = df['normal_loadshapes.csv']
+        enduse_loadshapes = df['enduse_loadshapes.csv']
         total_loadshapes = df['total_loadshapes.csv']
+        loadshapes = df['loadshapes.csv']
 
-        base_enduses = list(normal_loads.columns)
+        base_enduses = list(normal_loadshapes.columns)
         base_enduses.remove('time')
         base_enduses.remove('target')
         base_enduses.remove('daytype')
         base_enduses.remove('Heating')
         base_enduses.remove('Cooling')
-        
-        total_base_enduses = list(total_loadshapes.columns)
-        total_base_enduses.remove('time')
-        total_base_enduses.remove('target')
-        total_base_enduses.remove('daytype')
-        total_base_enduses.remove('Heating')
-        total_base_enduses.remove('Cooling')
-
         ticks = np.arange(0, 25, 3) 
 
         normal_plots_dir = f'{self.dir_name}/normal_loadshapes'
         self._create_results_storage(normal_plots_dir)
 
+        enduse_plots_dir = f'{self.dir_name}/enduse_loadshapes'
+        self._create_results_storage(enduse_plots_dir)
+
         total_plots_dir = f'{self.dir_name}/total_loadshapes'
         self._create_results_storage(total_plots_dir)
 
+        loadshapes_plots_dir = f'{self.dir_name}/loadshapes'
+        self._create_results_storage(loadshapes_plots_dir)
+
         logger.info('GENERATING NORMAL LOADSHAPE PLOTS')
 
         image_index = 0
-        for idx, city in enumerate(normal_loads.target.unique()):
-            city_df = normal_loads.loc[normal_loads.target == city]
-            
+        for idx, city in enumerate(normal_loadshapes.target.unique()):
+            city_df = normal_loadshapes.loc[normal_loadshapes.target == city]
+            max_total = city_df[['Heating', 'Cooling'] + base_enduses].sum(axis=1).max()
+            if max_total <= 1: max_val = 1
+            else: max_val = int(max_total) + 1
```

Comment:
you can change this to a one-liner: `max_val = 1 if max_total <= 1 else int(max_total) + 1`

Reply:
fixed

Reply:
I suggest we use
~~~
max_val = max(1.0,max_total)
~~~
so that the scale put max_total at the top when it exceeds 1.0, per our discussions with Joe.